os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -101,6 +101,25 @@ static void ns_printf(const char *input)
 	printf("%s\n", input);
 }
 
+
+
+#ifdef CONFIG_AMEBASMART_TRUSTZONE
+void *rtl_set_ns_func(void)
+{
+	/* No need for RTL8730E */
+	return NULL;
+}
+
+void up_allocate_secure_context(size_t size)
```

Comment:
Hi @sunghan-chang,

Currently, we have added up_allocate_secure_context() and up_free_secure_context() to allow SE to run without error.
In 8730E we are not using ARMV8M_TRUSTZONE, thus these 2 API is not present. However, RTL SE APIs called it, should we add a define switch to all the caller locations, or Samsung will help to add these 2 dummy APIs in TizenRT Kernel? If Samsung will help to add dummy APIs, these 2 dummy APIs in RTL SE can be removed.